bundle/src/main/java/com/adobe/acs/commons/data/Spreadsheet.java

```diff
@@ -124,6 +126,25 @@ public Spreadsheet(RequestParameter file, List<String> caseInsensitiveHeaders, S
         this(true, file, required);
         this.caseInsensitiveHeaders = caseInsensitiveHeaders;
     }
+    
+    /**
+     * Sets the locale to be used for numeric and date/time conversions.
+     * @param locale The new locale to use.
+     * @return This Spreadsheet instance for method chaining.
+     */
+    public Spreadsheet setLocale(Locale locale) {
```

Comment:
Instead of making this an explicit setter wouldn't it be better to just overload the method `buildSpreadsheet` with another version which takes the Locale? I don't think you actually need to store the Locale in a field and expose a getter for that. WDYT?

Reply:
Passing the locale through two layers of helper methods felt like a pain, but you're correct that the public API is cleaner if this is passed as a parameter instead of being set in a field.